test/specs/components/action.spec.jsx

```diff
@@ -242,24 +278,41 @@ xdescribe('Action', () => {
 
         describe('user has no email', () => {
             const props = getBuyProps();
+            const context = getContext({
+                app: {
+                    stripe: {
+                        appName: 'app-name',
+                        iconUrl: 'iconUrl'
+                    }
+                }
+            });
 
             beforeEach(() => {
 
+                appUtils.getIntegration.resolves({
```

Comment:
I feel like this is where mocking `getIntegration` hurts us. It's such a small function, all it does is filter an array.. why is it a util at all, and why does that deserve mocking?